Nos/Models/NoteParser.swift

```diff
@@ -6,6 +6,16 @@ import RegexBuilder
 /// This struct encapsulates the algorithms that parse notes and the mentions inside the note.
 struct NoteParser {
 
+    /// This struct encapsulates parsed components of a note that can be used to display the note in the UI.
```

Comment:
Nitpick:
```suggestion
    /// Encapsulates parsed components of a note that can be used to display the note in the UI.
```
I always try to avoid using the type in the doc comments. I usually start by writing it exactly how you did here, then I delete the part where I said "This class" or "This struct" or whatever. 😀

Reply:
Agree. I did that for consistency with the declaration of NoteParser right above it, but it's a different kind of thing. NoteParser is being used as a namespace here, and I think a protocol extension or caseless enum would be better for that.